syz-runner/runner.go

```diff
@@ -60,16 +62,35 @@ func main() {
 		Pool: rn.pool,
 		VM:   rn.vm,
 	}
-	if err := vrf.Call("Verifier.Connect", a, nil); err != nil {
+	r := &rpctype.RunnerConnectRes{}
+	if err := vrf.Call("Verifier.Connect", a, r); err != nil {
 		log.Fatalf("failed to connect to verifier: %v", err)
 	}
 
-	r := &rpctype.NextExchangeRes{}
-	if err := rn.vrf.Call("Verifier.NextExchange", &rpctype.NextExchangeArgs{Pool: rn.pool, VM: rn.vm}, r); err != nil {
+	if r.CheckUnsupportedCalls {
+		_, unsupported, err := host.DetectSupportedSyscalls(target, "none")
```

Comment:
Strictly speaking, the sandbox is determined by flagSandbox in ipcconfig. We may currently use only "none", but this is duplication that will fall apart when/if we pass sandbox specified in the config to syz-runner.
Extract the currently used sandbox from `config` variable.